examples/mnist/mnist.py

```diff
@@ -0,0 +1,62 @@
+import tensorflow as tf
+from input_data import read_data_sets
+from tfmesos import cluster
+from threading import Thread, RLock
+
+
+jobs_def = [
+    {
+        "name": "ps",
+        "num": 2
+    },
+    {
+        "name": "worker",
+        "num": 5
+    },
+]
+
+_lock = RLock()
+mnist = read_data_sets("MNIST_data/", one_hot=True)
+with cluster(jobs_def, master='cu1', quiet=False) as targets:
```

Comment:
It should be better to pass the mesos master address from something like `argv[1]`.